### PR TITLE
Fix on openExistingContact in Android and in example

### PR DESCRIPTION
**Pull request by engylemure**
@lukasgit  Hi Lukas, it seems that someone in another PR made some changes in how the internal arguments on the openExistingContacts work, but forgot too change it in Android.

### CODE REVIEW

**Comment by lukasgit** [comment]
@engylemure @lidongze91 is there a code conflict in previous PR's between the two of you? **Please confirm you have tested and are certain this PR should be merged.**

File: android/src/main/java/flutter/plugins/contactsservice/contactsservice/ContactsServicePlugin.java [hunk 1]
```diff
@@ -132,7 +132,7 @@ public void onMethodCall(MethodCall call, Result result) {
         }
         break;
       } case "openExistingContact" :{
-        final Contact contact = Contact.fromMap((HashMap)call.arguments);
+        final Contact contact = Contact.fromMap(((HashMap)call.argument("contact")));
```
Comment (lukasgit) [inline android/src/main/java/flutter/plugins/contactsservice/contactsservice/ContactsServicePlugin.java:135]: @engylemure why is this encapsulated in double parentheses?

Comment (engylemure) [reply android/src/main/java/flutter/plugins/contactsservice/contactsservice/ContactsServicePlugin.java:135]: @lukasgit it is not needed sorry for that.

**Comment by engylemure** [comment]
@lukasgit I'm sorry for the bug that was occurring in the Android, when i first developed these features (In the older flutter version) i had tested it in another project and was working just fine in a app that i was developing and the example. When i made the migration to the new version of flutter i had tested only in the example and no error or exceptions were occurring, so i considered that was working just fine, but it seems that was not the case. I had tested in my application and in the example seems that everything is working with these changes.

**Pull request merged**